tests/Agent/UnitTests/Core.UnitTest/Transformers/TransactionTransformer/ErrorTraceMakerTests.cs

```diff
@@ -24,6 +24,7 @@ public class ErrorTraceMakerTests
         private ErrorService _errorService;
         private AttributeDefinitionService _attribDefSvc;
         private IAttributeDefinitions _attribDefs => _attribDefSvc.AttributeDefs;
+        private const string _expectedErrorGroupAttributeName = "error.group.name";
```

Comment:
Same thing here (though both places might require that you initialize in the ctor rather than declare as a const here)